sampling.py

```diff
@@ -0,0 +1,84 @@
+import os
+import subprocess
+import sys
+
+from ruamel import yaml
+
+
+DEFAULT_CONFIG_PATH = 'config.yml'
+ASAPP_ROOT = os.environ['ASAPP_ROOT']
+ASAPP_PRODML_ROOT = os.environ['ASAPP_PRODML_ROOT']
```

Comment:
these can be pulled by importing `asapp.common.config` and calling `config.env_vars['ASAPP_COMCAST_SRS_ROOT']`

Reply:
Ah, much better. Thanks.